src/System.ValueTuple/src/System.ValueTuple.builds

```diff
@@ -6,6 +6,9 @@
     <Project Include="System.ValueTuple.csproj">
       <TargetGroup>portable-net40+sl4+win8+wp8</TargetGroup>
     </Project>
+    <Project Include="System.ValueTuple.csproj">
```

Comment:
I think we actually have a default somewhere that will set `NugetTargetMoniker` to netcoreapp1.0 if it is not set before running `PreReleaseResolveNugetPackageAssets`. Perhaps it's in buildtools? Anyway, as @weshaggard suggests, this would be an error case and would mean something is wrong with the project.